test/commands/contents.test.js

```diff
@@ -74,7 +75,66 @@ describe('contents command handler', () => {
 		);
 	});
 
-	test('<search> value not found, logs info message in console error', async () => {
+	test('when 3 repos are tested and a limit of 2 is specified, only 2 repos are logged', async () => {
+		const repositories = [
+			'Financial-Times/next-front-page',
+			'Financial-Times/next-signup',
+			'Financial-Times/n-gage'
+		];
+		const repositoriesForStdIn = repositories.join('\n');
+		standardInput = createStandardInput(repositoriesForStdIn);
+		repositories.forEach(repo => {
+			nockScope.get(`/${repo}/contents/Procfile`).reply(200, {
+				type: 'file',
+				content: 'd2ViOiBuLWNsdXN0ZXIgc2VydmVyL2luaXQuanM=', //base64 encoding of 'web: n-cluster server/init.js'
```

Comment:
You can use `base64Encode` to make this clearer

```suggestion
				content: base64Encode('web: n-cluster server/init.js'),
```

Same for the others